examples/collectives-polkadot/config.toml

```diff
@@ -33,10 +25,26 @@ cumulus_based = true
     name = "collator1"
     ws_port = 9710
     command = "./bin/polkadot-parachain"
-    args = [ "-lxcm=trace", "--state-cache-size=0" ]
+    args = ["-lxcm=trace"]
```

Comment:
Do not remove `--state-cache-size=0` it is essential to avoid having to add delays between actions making the tests more slow